pagopa/io-app
# fix: [IOBP-384] Header transaction summary safe area view

## Short description
This PR fixes the content top margin due to wrong SafeAreaView

## List of changes proposed in this pull request
- Moved `SafeAreaView` component one level upside;

## How to test
Try to pay any pagoPA notice until there is the transaction summary screen showed

## Preview
|Before|After|
|-|-|
|<img src="https://github.com/pagopa/io-app/assets/34343582/7165fb4a-3782-4435-9b18-c8b0f3963e3b" width="300" />|<img src="https://github.com/pagopa/io-app/assets/34343582/438c5d21-d310-4e8a-83d4-0051c85483d7" width="300" />
